src/redux/data.js

```diff
@@ -371,6 +372,21 @@ const checkForConfirmedSavingsActions = transactionsData => dispatch => {
   }
 };
 
+const checkForUpdatedNonce = transactionData => dispatch => {
+  const [latestTx] = transactionData;
```

Comment:
it is rare but zerion may append multiple txns at once - we may want to include the same txSortedByAscendingNonce logic here